perspective.el

```diff
@@ -164,6 +174,18 @@ Run with the perspective to be destroyed as `persp-curr'.")
   "A hook that's run after a perspective has been activated.
 Run with the activated perspective active.")
 
+(defvar persp-state-before-save-hook nil
+  "A hook run immediately before saving persp state to disk.")
+
+(defvar persp-state-saved-hook nil
+  "A hook run immediately after saving persp state to disk.")
+
+(defvar persp-state-before-load-hook nil
+  "A hook run immediately before loading persp state from disk.")
+
+(defvar persp-state-loaded-hook nil
+  "A hook run immediately after loading persp state from disk.")
```

Comment:
Maybe `after-save` and `after-load`? (I'm biased towards symmetry)

Reply:
Agreed: https://github.com/gcv/perspective-el/commit/ff7f03709f4edf445b841aff75cfad46dfd8da08